### PR TITLE
DF-2204 - Creates time macro, breaks apart format macros

### PR DESCRIPTION
**Pull request by anselmbradford**
Creates time macro and converts datetime instances to use time macro. Also, breaks apart format macro into topical macros.

## Additions

- Adds time macro.

## Changes

- Converted time elements to use time template.
- Broke apart format macros into topical macros.

## Testing

- `/activity-log/`, `/events/`, `/events/summer-consumer-advisory-board-meeting-in-omaha-ne/`, `/blog/`, `/blog/forum-on-know-before-you-owe-eclosing-pilot/`, `/the-bureau/leadership-calendar/`, `/offices/payments-to-harmed-consumers/`, `/careers/`, `/careers/current-openings/`, `/careers/current-openings/1873/`

## Review

- @sebworks 
- @KimberlyMunoz 
- @schaferjh 

## Screenshots

On beta:
screen shot 2015-08-11 at 10 37 40 am

This PR:
screen shot 2015-08-11 at 10 37 25 am

---

On beta:
screen shot 2015-08-11 at 10 39 53 am

This PR:
screen shot 2015-08-11 at 10 39 28 am

---

On beta:
screen shot 2015-08-11 at 10 40 59 am

This PR:
screen shot 2015-08-11 at 10 40 37 am

---

On beta:
screen shot 2015-08-11 at 10 41 59 am

This PR:
screen shot 2015-08-11 at 10 41 52 am

---

On beta:
screen shot 2015-08-11 at 10 42 43 am

This PR:
screen shot 2015-08-11 at 10 42 32 am

---

On beta:
screen shot 2015-08-11 at 10 43 35 am

This PR:
screen shot 2015-08-11 at 10 43 29 am

---

On beta:
screen shot 2015-08-11 at 10 44 50 am

This PR:
screen shot 2015-08-11 at 10 44 42 am

---

On beta:
screen shot 2015-08-11 at 10 45 34 am

This PR:
screen shot 2015-08-11 at 10 45 27 am

---

On beta:
screen shot 2015-08-11 at 10 46 19 am

This PR:
screen shot 2015-08-11 at 10 46 14 am

---

On beta:
screen shot 2015-08-11 at 10 47 04 am

This PR:
screen shot 2015-08-11 at 10 46 59 am

---

On beta:
[JOB POSTING NOT AVAILABLE ON BETA]

This PR:
screen shot 2015-08-11 at 10 47 39 am

## Todos

- The dates on `/the-bureau/history/` are not fed into the macro. Should they be?
screen shot 2015-08-11 at 10 44 19 am 
- Should the dates all be uppercase? @duelj

### CODE REVIEW

File: src/_includes/macros/time.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    {% if show_only.date == true and show_only.time == true %}
+    <span class="datetime_divider">@</span>
+    {% endif %}
+
+    {% if show_only.time == true %}
+    <span class="datetime_time">
+        {% if show_only.timezone == true %}
+        {{ format_time(datetime, timezone) }}</span>
+        {% else %}
+        {{ format_time(datetime) }}</span>
```
Comment (sebworks) [inline src/_includes/macros/time.html]: Shouldn't closing tag be outside of if statement?

Comment (anselmbradford) [inline src/_includes/macros/time.html]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/b10968cfc5a82eb26553fae59c30d1b218181314 Also, the divider shouldn't have been enclosed in the time element.

File: src/_includes/article.html [hunk 2]
```diff
@@ -61,8 +61,9 @@ <h1 class="post_heading">
                 &ndash;
             </span>
         {% endif %}
-            <span class="post_date date">
-                {{ post.date|date("%b %d, %Y") }}
+            <span class="date">
+                {% import "macros/time.html" as time %}
```
Comment (KimberlyMunoz) [inline src/_includes/article.html]: Can you use jinja single quotes here and throughout?

Comment (anselmbradford) [inline src/_includes/article.html]: Doh! Sorry, will do.

**Comment by KimberlyMunoz** [comment]
I'm going through to check how they render on all these and the attention to detail on all these `<time>`s is fantastic.

**Comment by duelj** [comment]
The example shown look great. I would wait on the history example since we have a revised design for that page.

File: src/_includes/article.html (hunk 2 above)
Comment (anselmbradford) [inline src/_includes/article.html]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/5cdae78991a4cb487fbe2288b7fbb10132e9a955 Let me know if I missed any.

**Comment by KimberlyMunoz** [comment]
:+1:

**Comment by schaferjh** [comment]
looks good!

**Pull request merged**